### PR TITLE
Header and Footer Cleanups

### LINKED ISSUE #86: Handle the secondary nav on mobile
- [ ] Don't have two hamburgers

 
Image

### LINKED ISSUE #101: Cohesive header with xyz
As we want both the xyz and docs site to feel as 1 site we need to align the header content to be such. 
See image below to see alignment issues (_top is xyz, bottom is docs site_)

Screenshot 2024-11-13 at 1 15 35 PM

### LINKED ISSUE #104: Prev/next chapter buttons look strange when in mobile and end up stacking

### PR DESCRIPTION
**Pull request by wilwade**
Problem
=======

Various small layout issues addressed (one of a few PRs for this sort of stuff)

Closes #101
Closes #104 
Closes #86
Part of #87

Solution
========
- Fix font issue with Bold
- Remove dead styles
- Fix content margins to handle the left nav buttons with centering
- Fix the previous/next buttons on mobile
- Add a margin to the footer to fix the footer border from hitting the side nav (slight design break)

Screenshots (optional):
-----------------------
image

image

image

image

### CODE REVIEW

File: theme/css/general.css [hunk 1]
```diff
@@ -155,15 +155,7 @@ h6:target::before {
 
 .content {
   overflow-y: auto;
-  padding: 30px 15px;
```
Comment (wilwade) [inline theme/css/general.css:158]: Moved this to tailwind classes to easily handle responsive breaks

File: theme/css/tailwind.css [hunk 2]
```diff
@@ -69,22 +69,6 @@
     @apply opacity-100;
     transform: translate3d(0, 0, 0);
   }
-
-  /*  Form common styles*/
```
Comment (wilwade) [inline theme/css/tailwind.css:73]: No form styles needed

File: theme/css/tailwind.css [hunk 3]
```diff
@@ -97,6 +81,11 @@
     margin: 0.67em 0;
   }
 
+  /* Reset the margins for the first header in the contents */
+  .content h1:first-of-type {
```
Comment (wilwade) [inline theme/css/tailwind.css:85]: Moved to be closer to the thing it is modifying

File: theme/index.hbs [hunk 4]
```diff
@@ -202,21 +202,21 @@
                 });
             </script>
 
-            <div id="content" class="content">
+            <div id="content" class="content pt-8 pb-32 pl-[6px] pr-[45px]">
```
Comment (wilwade) [inline theme/index.hbs:205]: Handles the left nav buttons better

File: theme/index.hbs [hunk 5]
```diff
@@ -202,21 +202,21 @@
...
                 <main>
                     {{{ content }}}
                 </main>
 
-                <nav class="nav-wrapper" aria-label="Page navigation">
+                <nav class="nav-wrapper p-6" aria-label="Page navigation">
                     <!-- Mobile navigation buttons -->
                     {{#previous}}
-                        <a rel="prev" href="{{ path_to_root }}{{link}}" class="mobile-nav-chapters previous" title="Previous chapter" aria-label="Previous chapter: {{title}}" aria-keyshortcuts="Left">
+                        <a rel="prev" href="{{ path_to_root }}{{link}}" class="mobile-nav-chapters previous mb-6 w-full md:w-auto" title="Previous chapter" aria-label="Previous chapter: {{title}}" aria-keyshortcuts="Left">
```
Comment (wilwade) [inline theme/index.hbs:213]: Go full width on mobile and margins for stacking

File: theme/index.hbs [hunk 6]
```diff
@@ -227,7 +227,7 @@
         </div>
 
         <!-- Custom Footer Start -->
-        <footer id="footer" class="bg-white px-6 md:px-[10%] max-w-[100vw]">
+        <footer id="footer" class="bg-white pt-6 px-6 md:px-[10%] max-w-[100vw]">
```
Comment (wilwade) [inline theme/index.hbs:230]: Slight top padding to separate the nav bar from the footer border

**Review by saraswatpuneet — APPROVED** [review]
Tried it locally, looks good to me

**Pull request merged**